src/platform-includes/configuration/http-client/javascript.mdx

```diff
@@ -0,0 +1,62 @@
+```javascript {tabTitle: JavaScript}
+import * as Sentry from "@sentry/browser";
+import { HttpClient as HttpClientIntegration } from "@sentry/integrations";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [
+    new HttpClientIntegration({
```

Comment:
Looking at this again, I am wondering if we should have made the default [400, 599] instead. What was our thought process here again?

Reply:
`[500, 599]` is defined in the spec: 
https://develop.sentry.dev/sdk/features/#http-client-errors

If I remember correctly, this also came up a few times as issues in Remix and Next SDKs (their request auto wrappers). It seems users don't normally expect 4xx's on their dashboard.